codegenerator/cli/templates/dynamic/contract_import_templates/shared/.env.hbs

```diff
@@ -0,0 +1,7 @@
+#To create or update a token visit https://envio.dev/app/api-tokens
+{{#if hypersync_api_token}}
+HYPERSYNC_API_TOKEN="{{hypersync_api_token}}"
+{{else}}
+#Uncomment the line below and set a valid token
+#HYPERSYNC_API_TOKEN="<YOUR-API-TOKEN>"
+{{/if}}
```

Comment:
1. As we discussed I'd like to start prefixing env variables with `ENVIO`
2. I think the fact that it's a HyperSync token is an internal knowledge and might only confuse some users. So `ENVIO_API_TOKEN` is a better choice in my opinion. Also, it'll stay up to date, when we add organisation specific tokens specifically for indexer, or something like this. 
3. Having a space after `#` in a comment looks better to me

```suggestion
# To create or update a token visit https://envio.dev/app/api-tokens
{{#if hypersync_api_token}}
ENVIO_API_TOKEN="{{hypersync_api_token}}"
{{else}}
# Uncomment the line below and set a valid token
# ENVIO_API_TOKEN="<YOUR-API-TOKEN>"
{{/if}}
```

Reply:
Cool, sounds good 👍🏼